Update README.md

felixsuarez0727 opened the pull request:
Update README

This pull request updates the README file to include the most relevant information.

hammal reviewed (commented):
Great work.

Two small comments.
- it should read control-bounded ADCs (CBADCs)
- I would suggest oversampling ration (OSR) (instead of osr).

felixsuarez0727 commented:
> Great work.
> 
> Two small comments.
> 
> * it should read control-bounded ADCs (CBADCs)
> * I would suggest oversampling ration (OSR) (instead of osr).

Thanks for the feedback! I've updated the text to use 'control-bounded ADCs (CBADCs)' and changed 'osr' to 'oversampling ratio (OSR)' as suggested. Please let me know if there's anything else that needs adjustment!
